gap/PolygonalComplexes/modification.gd

```diff
@@ -576,6 +576,17 @@ DeclareOperation( "SplitEdgePathNC", [IsPolygonalComplex, IsVertexEdgePath and I
 #! The NC-version does not check whether the given set of <A>faces</A>
 #! actually consists only of faces in <A>complex</A>. It also does not
 #! check whether the result of <K>SubsurfaceByFaces</K> is a surface.
+#!
+#! In Chapter <Ref Chap="Chapter_EdgeColouring"/> edge colouring will be introduced.
+#! If we compute a subcomplex of an edge-coloured complex, it will be edge-coloured again,

```

Comment:
Maybe put this in another example session in order to separate lines 584-586 from 587 - 588

If P is an edge coloured, then the complex that is computed by the function SubsurfaceByFaces with a given set of faces is a polygonal complex with an edge colouring that is induced by the colouring of P